doc/rados/operations/health-checks.rst

```diff
@@ -499,13 +499,18 @@ Filestore OSDs can be listed with:
 
    ceph report | jq -c '."osd_metadata" | .[] | select(.osd_objectstore | contains("filestore")) | {id, osd_objectstore}'
 
-If it is not feasible to migrate Filestore OSDs to Bluestore immediately, you
+In order to upgrade to Reef or later releases, any Filestore OSDs must first be
+migrated to BlueStore.
+For releases before Reef, If it is not feasible to migrate Filestore OSDs to BlueStore immediately, you
```

Comment:
sFor releases before Reef, If it is not feasible to migrate Filestore OSDs to BlueStore immediately, you/For releases before Reef: if it is not feasible to migrate Filestore OSDs to BlueStore immediately, you/

That substitution statement is a mere suggestion. What's more than a mere suggestion though is that "If" should be "if".

Reply:
Ack.  Pushing revision.  Failure to change the I when prepending to the sentence.